images/tkg/README.md

```diff
@@ -92,14 +92,14 @@ make clean IMAGE_ARTIFACTS_PATH=/root/artifacts/ LABEL=byoi_image_builder # To c
 
 - `make run-artifacts-container` is used to run the artifacts container for a Kubernetes version at a particular port
   - artifacts image URL will be fetched from the [supported-versions.json](supported-versions.json) based on the Kubernetes version selected.
-  - By default artifacts container will use 8080 port but this can be configured through `ARTIFACTS_CONTAINER_PORT` parameter.
+  - By default artifacts container will use an 8080 port but this can be configured through the `ARTIFACTS_CONTAINER_PORT` parameter.
```

Comment:
maybe

By default artifacts container uses port 8080 by default however this can be configured through the `ARTIFACTS_CONTAINER_PORT` parameter.